packages/node_modules/@ciscospark/plugin-board/test/integration/spec/board.js

```diff
@@ -179,44 +179,54 @@ describe(`plugin-board`, () => {
       });
 
       it(`retrieves all boards for a specified conversation across multiple pages`, () => {
-        let existingChannelsCount = 0;
-        let numChannelsToAdd = 0;
-        const pageLimit = 50;
+        const numChannelsToAdd = 12;
+        const pageLimit = 5;
+        const channelsCreated = [];
+        let channelsReceived = [];
+        let convo;
 
-        return participants[0].spark.board.getChannels(conversation, {
-          channelsLimit: 100
+        return participants[0].spark.conversation.create({
+          displayName: `Test Get Channels Conversation ${uuid.v4()}`,
+          participants
         })
-          .then((res) => {
-            existingChannelsCount = res.length;
-            assert(existingChannelsCount < pageLimit, `the existing channel count must be less that pageLimit for this test`);
-            numChannelsToAdd = pageLimit - existingChannelsCount + 1;
-          })
-
-          .then(() => {
+          .then((c) => {
+            convo = c;
             const promises = [];
 
             for (let i = 0; i < numChannelsToAdd; i++) {
-              promises.push(participants[0].spark.board.createChannel(conversation));
+              promises.push(participants[0].spark.board.createChannel(convo)
+                .then((channel) => {
+                  Reflect.deleteProperty(channel, `kmsMessage`);
+                  channelsCreated.push(channel);
+                }));
             }
             return Promise.all(promises);
           })
-
           // get boards, page 1
           .then(() => {
-            return participants[0].spark.board.getChannels(conversation, {
+            return participants[0].spark.board.getChannels(convo, {
               channelsLimit: pageLimit
             });
           })
-
           // get boards, page 2
           .then((channelPage) => {
             assert.lengthOf(channelPage.items, pageLimit);
             assert(channelPage.hasNext());
+            channelsReceived = channelsReceived.concat(channelPage.items);
+            return channelPage.next();
+          })
+          // get boards, page 3
+          .then((channelPage) => {
+            assert.lengthOf(channelPage.items, pageLimit);
+            assert(channelPage.hasNext());
+            channelsReceived = channelsReceived.concat(channelPage.items);
             return channelPage.next();
           })
           .then((channelPage) => {
-            assert.lengthOf(channelPage, 1);
+            assert.lengthOf(channelPage, 2);
             assert(!channelPage.hasNext());
+            channelsReceived = channelsReceived.concat(channelPage.items);
+            assert.sameDeepMembers(channelsCreated, channelsReceived);
```

Comment:
does sameDeepMembers allow for two arrays to have the same set of items, but in a different order?